leptos_dom/src/html.rs

```diff
@@ -651,8 +677,14 @@ impl<El: ElementDescriptor + 'static> HtmlElement<El> {
         }
     }
 
-    /// Adds multiple attributes to the element
+    /// Adds multiple attributes to the element.
     #[track_caller]
+    #[deprecated(
```

Comment:
I try to avoid adding `#[deprecated]` in patch releases because it can break CI/CD for people if they are set up to deny on warnings and their build pulls in a newer patch version than they have locally.

Reply:
Removed it.